chef/cookbooks/cpe_utils/libraries/node_functions.rb

```diff
@@ -356,5 +368,29 @@ def installed?(bundle_identifier)
       paths = app_paths(bundle_identifier)
       !paths.empty?
     end
+
+    def min_package_installed?(pkg_identifier, min_pkg)
+      unless macos?
+        Chef::Log.warn('node.min_package_installed? called on non-OS X!')
+        return false
+      end
+      installed_pkg_version = shell_out(
+        "/usr/sbin/pkgutil --pkg-info \"#{pkg_identifier}\"",
+      ).run_command.stdout.to_s[/version: (.*)/, 1]
```

Comment:
on the off chance, this is nil, line 381 will blow up

`chef (13.8.5)> Gem::Version.new(nil) >= Gem::Version.new('1.0') 
 => false 
`

Reply:
Fixed